site/content/docs/cli/constructs-reference.md

```diff
@@ -361,7 +361,7 @@ new ApiCheck('check-group-api-check-1', {
 - `muted`: A boolean value if alert notifications from the Checks in the group are muted, i.e. not sent out.
 - `tags`: An array of tags. Group tags trickle down to tags on the individual Checks. i.e. `['product', 'api']`
 - `runtimeId`: The ID of which [runtime](/docs/runtimes/specs/) to use for the Checks in the group.
-- `environmentVariables`: An array of objects defining variables in the group scope, i.e. `[{ key: 'DEBUG', value: 'true' }]`
+- `environmentVariables`: An array of objects defining variables in the group scope, i.e. `[{ key: 'DEBUG', value: 'true', secret: 'true' | locked: 'true' }]`
```

Comment:
uhm, does this agree with what we discussed? 😓

Reply:
@MariadeAnton how do you mean?

Reply:
ah! just the CLI variable name `environmentVariables` does includes the word "environment" in it :)

Reply:
Gotcha, yea we don't want to change the CLI too much, it could break scripts, GH action workflows etc.